samples/samples-python/AddProductParams/__init__.py

```diff
@@ -1,11 +1,15 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+from urllib.parse import parse_qs, urlparse
+
 import azure.functions as func
 from Common.product import Product
 
 def main(req: func.HttpRequest, product: func.Out[func.SqlRow]) -> func.HttpResponse:
-    row = func.SqlRow(Product(req.params["productId"], req.params["name"], req.params["cost"]))
+    # The Python worker discards empty query parameters so use parse_qs as a workaround.
```

Comment:
Should we document this somewhere or is this just temporary (is the Python worker going to change this behavior?)

Reply:
Waiting on the Python worker team's response on the issue https://github.com/Azure/azure-functions-python-worker/issues/894